crates/lib/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+pub mod filters;
+pub mod partials;
+pub mod tags;
+
+#[cfg(feature = "all")]
+pub use crate::filters::extra::*;
+#[cfg(feature = "jekyll")]
+pub use crate::filters::jekyll::*;
+#[cfg(feature = "stdlib")]
+pub use crate::filters::std::*;
+
+pub use crate::tags::*;
```

Comment:
`tags `would be a part of `stdlib`.

For example, the `include` tag is actually the jekyll version.  We;ll eventually need to move that under jekyll and also implement the stdlib version.